pages/api/search/[fragment].ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios"
+
+const BASE_URL = "https://cloud.iexapis.com"
+const TOKEN = "pk_63509e5b43384ab08845be28759fb5ea"
+
+const fetcher = async (fragment) => {
+  try {
+    const { data } = await axios.get(`${BASE_URL}/stable/search/${fragment}?token=${TOKEN}`)
+    return data
+  } catch (error) {
+    console.error(error)
+    return {}
+  }
+}
+
+export default (req, res) => {
+  const { fragment } = req.query
+  fetcher(fragment)
+    .then((data) => {
+      res.status(200).json(data)
+    })
+    .catch(() => {
```

Comment:
I think right now this block will never run, because your fetcher will swallow the error.    Eventually we could think through all the different error states (which we would define as 400s and 500s), etc.